accord-core/src/main/java/accord/coordinate/Agree.java

```diff
@@ -45,11 +49,76 @@ public boolean hasMetFastPathCriteria()
         }
     }
 
+    static class PreacceptTracker extends FastPathTracker<ShardTracker>
+    {
+        volatile long supersedingEpoch = -1;
```

Comment:
I assume this is volatile so the configuration service (which is unrelated to the `Instance`) may invoke directly, but would it be easier to ask the `Instance` to perform epoch work so we can avoid worrying about the concurrency?